### ISSUE TITLE
ENOENT when running "Factorio: Select Version"

### ISSUE THREAD

**Issue by Cooldude2606** [justarandomgeek/vscode-factoriomod-debug#162]
TL;DR when running "Factorio: Select Version" from the command pallet i get the following error:

![image](https://github.com/user-attachments/assets/ee3ac5fb-5cf2-4d5f-90ed-bdf313ed6897)

There are no factorio versions defined in the workspace settings, and my user settings only contain one version which is the default steam path for factorio `C:\Program Files (x86)\Steam\steamapps\common\Factorio\bin\x64\factorio.exe`

The path `C:\Program Files\Factorio` does exist and is a `0.16.51` install of factorio, the `doc-html` folder does not include any json files.

Note: I am not attempting to select this `0.16` version, I am unable to open the menu to select a version, I assume because it is attempting to scan default locations for factorio.

**Comment by justarandomgeek** [comment]
should be fixed in 2.0.6
